src/components/Verification/Verification.tsx

```diff
@@ -69,13 +76,26 @@ const Verification = () => {
     };
 
     const checklistItems = [
+        ...(isPhoneNumberVerificationEnabled
+            ? [
+                  {
+                      onClick: () => {
+                          window.location.href = `${URLConstants.derivAppProduction}/account/personal-details`;
```

Comment:
can we do the redirection based on the environment.. if test link or staging, redirect to staging-app.deriv. else app.deriv ?

Reply:
hmmm i dont think its necessary tbh, because either way we can still test the PNV using just prod link. Also even if we use staging and so forth, QA still need to set up the endpoint again